metaflow/main_cli.py

```diff
@@ -349,83 +348,79 @@ def sandbox(profile):
     # Persist to a file.
     persist_env(env_dict, profile)
 
-@configure.command(help='Get Metaflow up and running on your own AWS environment.')
+@configure.command(help='Configure Metaflow to access self-managed AWS resources.')
```

Comment:
Any reason you chose against the 'Get Metaflow up and running on your self-managed AWS resources.'? Is it getting trimmed?

Currently the language looks slightly inconsistent between `sandbox` and `aws`:
```
Commands:
  aws      Configure Metaflow to access self-managed AWS resources.
  reset    Reset configuration to run locally.
  sandbox  Get Metaflow up and running on our sandbox.
  show     Show existing configuration.
```

Reply:
Given that all the statements start with an assertive verb, it made sense to modify `Get...` to `Configure...`. I will make a similar language change for sandbox.